examples/multi-node-training/README.md

```diff
@@ -44,8 +46,26 @@ docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_va
 
 > For AWS DL1 instances, `--privileged` must be passed to the `docker run` command so that EFA interfaces are visible.
 
-Finally, you will have to copy the public key of the leader node in the `~/.ssh/authorized_keys` file of all other nodes to enable password-less SSH.
 
+You will have to copy id_rsa.pub contents from every node's docker to every other node's docker's ~/.ssh/authorized_keys (all public keys need to be in all hosts' authorized_keys) to enable password-less SSH.
```

Comment:
Weird I never had to do that on AWS DL1 instances with EFA

Reply:
Created EFA folder and move AWS dockerfile and deepspeed_env to there. 
But this step is mentioned previously also and I just explained step by step details.

Reply:
Yes but in my understanding we only need to copy the key of the leader node to every other bode's authorized keys, not every key.

Reply:
I had to follow the step from https://github.com/HabanaAI/Model-References/tree/master/PyTorch/nlp/DeepSpeedExamples/deepspeed-bert#set-up-password-less-ssh for GaudiNIC.

Let me try just leader node key also.

Reply:
I confirmed that we only need to copy leader node's key. I updated the README.